src/stravalib/tests/integration/test_client.py

```diff
@@ -1082,3 +1109,60 @@ def test_get_activity_kudos(mock_strava_api, client):
     assert isinstance(kudoer_list[0], SummaryAthlete)
     assert len(kudoer_list) == 2
     assert kudoer_list[0].lastname == "Doe"
+
+
+@patch("stravalib.protocol.ApiV3.exchange_code_for_token")
+def test_exchange_code_for_token_athlete(
+    mock_request, client, raw_exchange_response
+):
+    """If a user requests athlete data, then it should be returned as a
+    SummaryAthlete object."""
+    mock_request.return_value = raw_exchange_response
+
+    result = client.exchange_code_for_token(
```

Comment:
Can you also try to unpack the `result` tuple here (and adapt the assertions below) to make the test more explicit?

Reply:
Ahhhh. I really appreciate this feedback. Thank you! This test is much cleaner / better now as is the code. I was trying to avoid additional conditional statements in the methods, but it seems by doing that I added complexity and less specificity to the tests and code.